pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 79
```

Comment:
We've been using 99 for code actually since 79 was super constraining and everything went over multiple lines or variable names became unreadable.

Reply:
and with 99 it doesn't fit on the monitor to do side by side reviews or to have an interactive interpreter beside the code window :D This doesn't change the flake check, just for the future it'd format the code shorter. Also, we used to have long variable names, but they have become more python like and much shorter, and now it's much easier to fit the code in 80 chars.

Reply:
Yeah: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

I think we should keep 79 here; leave the flake8 at 99 for the time being, but try to thoughtfully shorten variable names as we touch the relevant parts of code.

Reply:
There's definitely a lot to be said for going with defaults (79 in this case), and if it doesn't require us to adjust everything right away I don't mind at all. I'll just adjust from now on 😆